File: CreativCyborgApp/app/src/main/java/com/example/creativcyborg/gateway/dto/ingoing/RoomContentDTO.java
package com.example.creativcyborg.gateway.dto.ingoing;

import com.example.creativcyborg.entities.ContentPart;

import java.util.List;

public class RoomContentDTO
{
    public List<ContentPart> content;
    public int audioFrequenz;

    public RoomContentDTO(List<ContentPart> content, int audioFreq)
    {
        content = content;
        audioFrequenz = audioFreq;
    }
}

File: CreativeCyborgApi/CreativeCyborgApi/Boundary/DTO/RoomContentDTO.cs
using CreativeCyborgApi.Entity;

namespace CreativeCyborgApi.Boundary.DTO
{
    public class RoomContentDTO
    {
        public List<ContentPart> Content { get; set; }
        public int AudioFrequenz { get; set; }

        public RoomContentDTO(List<ContentPart> content, int audioFreq) 
        {
            Content = content;
            AudioFrequenz = audioFreq;
        }
    }
}
